public/app/features/dashboard-scene/saving/DashboardSceneChangeTracker.ts

```diff
@@ -117,6 +126,36 @@ export class DashboardSceneChangeTracker {
     if (isSceneVariableInstance(payload.changedObject)) {
       return true;
     }
+
+    if (payload.changedObject instanceof RowsLayoutManager) {
+      return true;
+    }
+    if (payload.changedObject instanceof RowItem) {
+      return true;
+    }
+    if (payload.changedObject instanceof TabsLayoutManager) {
+      return true;
+    }
+    if (payload.changedObject instanceof TabItem) {
+      return true;
+    }
+    if (payload.changedObject instanceof AutoGridLayoutManager) {
+      return true;
+    }
```

Comment:
ResponsiveGridItem is needed as well is that already added?

Reply:
Initially thought that wouldn't be necessary as the auto grid items don't have any position info, like the other grid items.

But I realized now that it is still what keeps the repeat options. Will add it too.